src/devices/hzc_electric.ts

```diff
@@ -53,6 +53,17 @@ const definitions: DefinitionWithExtend[] = [
         description: 'Zigbee dimmer',
         extend: [light({configureReporting: true})],
     },
+    {
+        zigbeeModel: ['Meter-Dimmer-Switch-ZB3.0'],
+        model: 'Meter-Dimmer-Switch-ZB3.0',
```

Comment:
is there a model number written on the device or product page?

Reply:
Yes, on the device it says D692-ZG. But to Zigbee2Mqtt it is reported as "Meter-Dimmer-Switch-ZB3.0"

Reply:
Yes, could you update this PR?

Reply:
It's updated!